### PR TITLE
OpenEXR: bump

### PR DESCRIPTION
**Pull request by extrowerk**
Castrated old, tools, fixed includes and some cleanups

### CODE REVIEW

**Comment by korli** [comment]
see https://git.haiku-os.org/haiku/tree/build/jam/repositories/HaikuPorts/x86_gcc2#n123
and https://git.haiku-os.org/haiku/tree/build/jam/repositories/HaikuPorts/x86_gcc2#n253

**Comment by extrowerk** [comment]
It means i should keep the listed versions?

**Comment by korli** [comment]
I would add a openexr25-2.5.4.recipe and leave the rest as is.

File: media-libs/openexr/openexr25-2.5.4.recipe [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+	cmd:exrenvmap$secondaryArchSuffix = $portVersion
+	cmd:exrheader$secondaryArchSuffix = $portVersion
+	cmd:exrmakepreview$secondaryArchSuffix = $portVersion
+	cmd:exrmaketiled$secondaryArchSuffix = $portVersion
+	cmd:exrmultipart$secondaryArchSuffix = $portVersion
+	cmd:exrmultiview$secondaryArchSuffix = $portVersion
+	cmd:exrstdattr$secondaryArchSuffix = $portVersion
+	"
+REQUIRES_tools="
+	haiku$secondaryArchSuffix
```
Comment (korli) [inline media-libs/openexr/openexr25-2.5.4.recipe:54]: Already in REQUIRES.

**Pull request merged**